.circleci/config.yml

```diff
@@ -785,7 +785,7 @@ jobs:
   release-checks:
     <<: *base-job
     # Fix-me: Carthage can't build fat frameworks on Apple Silicon. See https://github.com/RevenueCat/purchases-ios/pull/3582
-    resource_class: macos.x86.medium.gen2
+    resource_class: macos.m1.medium.gen1
```

Comment:
We should remove all these comments if it works (I imagine we might need to do some changes?)